### ISSUE TITLE
Mark objects that use both addr:street=... and addr:place=... tagging

### ISSUE THREAD

**Issue by ltog** [ltog/osmi-addresses#103]

**Comment by stephan75** [comment]
+1 !!
If you want to verify any of your results, you can use the wizard mode of overpass-turbo.eu and type "addr:street"=* and "addr:place"=* ... or not?

**Comment by ltog** [comment]
@stephan75 

> you can use the wizard mode of overpass-turbo.eu

Yes in principle that should work. (I guess it needs a bit of juggling with Overpass syntax and I'm not sure about the performance...)

If you're fluent with Overpass I wouldn't mind if you would paste the corresponding code here... ;-)

But I also assume verifying won't be needed so much here since I expect that once this function basically works, there is not much that can go wrong. But of course it will need some time to implement and deploy.

I also had this idea to start another project where we collect samples of Overpass code snippets which can be used for OSM quality assurance. The advantage would be that such checks wouldn't need a continually maintained server part (except for the Overpass server of course). But at the moment this really is just an idea...

**Comment by stephan75** [comment]
> I also had this idea to start another project where we collect samples of Overpass code snippets which can be used for OSM quality assurance.

Too late:  ;-)

see http://wiki.openstreetmap.org/wiki/DE:Overpass_API/Beispielsammlung
or http://wiki.openstreetmap.org/wiki/Overpass_API/Overpass_API_by_Example
or http://wiki.openstreetmap.org/wiki/Overpass_turbo/Examples

**Comment by ltog** [comment]
@stephan75 : Thanks for the links. Some of the examples are already useful for QA...
